source/sdr.rst

```diff
@@ -471,32 +469,6 @@ The simplest categorization of LULCs on the landscape involves delineation by la
 
 A slightly more sophisticated LULC classification involves breaking relevant LULC types into more meaningful types. For example, agricultural land classes could be broken up into different crop types or forest could be broken up into specific species. The categorization of land use types depends on the model and how much data is available for each of the land types. You should only break up a land use type if it will provide more accuracy in modeling. For instance, only break up 'crops' into different crop types if you have information on the difference in USLE C values between crops.
 
-*Sample Land Use/Land Cover Table*
-
-  ====== ===========================
-  lucode Land Use/Land Cover
-  ====== ===========================
-  1      Evergreen Needleleaf Forest

```

Comment:
Ah, interesting. This is annoyingly redundant. I wonder, outside of it's own section like I mentioned above, if this is a case where having a CSV stored in the UG repo makes sense, so it can be re-used in places like this? 

Unless we decide this isn't helpful at all.